resolved PR comments

ShreeJithR opened the pull request:
- Bulk upload test case enhancements as per PR comments
- Added description to few of the test cases

ShreeJithR committed:
resolved PR comments

- Bulk upload test case enhancements as per PR comments
- Added description to few of the test cases

infa-rvarahag commented on src/test/java/org/shikshalokam/backend/MentorEDSessionUpdate.java:
```diff
@@ -0,0 +1,143 @@
+                            .header("X-auth-token", "bearer " + X_AUTH_TOKEN)
+                            .contentType("application/json")
+                            .body(jsonBody)
+                            .post(new URI(sessionUpdateEndPoint));
```
JSONObject jsonObject = (JSONObject) new JSONParser().parse(new FileReader(filePath));

infa-rvarahag commented on src/test/java/org/shikshalokam/backend/MentorEDSessionUpdate.java:
```diff
@@ -0,0 +1,143 @@
+                    Response response = RestAssured.given()
+                            .header("X-auth-token", "bearer " + X_AUTH_TOKEN)
+                            .contentType("application/json")
+                            .body(jsonBody)
```
.body(jsonObject)

infa-rvarahag commented on src/test/java/org/shikshalokam/backend/MentorEDSessionUpdate.java:
```diff
@@ -0,0 +1,143 @@
+            List<String> updatedCSVLines = IntStream.range(0, csvLines.size())
+                    .mapToObj(i -> csvLines.get(i).contains("SessionId") && i < sessionIds.size()
+                            ? csvLines.get(i).replace("SessionId", sessionIds.get(i))
+                            : csvLines.get(i))
```
content = content.replace(placeholderName, replacementName)
                             .replace(placeholderCity, replacementCity)

ShreeJithR committed:
enhancement

changes
1 .body(jsonBody) > .body(jsonObject)
2 JSONObject jsonObject = (JSONObject) new JSONParser().parse(new FileReader(jsonFilePath));
3   content = content.replaceFirst("SessionId", sessionId);

ShreeJithR committed:
Update MentorEDSessionUpdate.java
